src/lib/duration/iso-string.js

```diff
@@ -21,7 +21,7 @@ export function toISOString() {
     // 3600 seconds -> 60 minutes -> 1 hour
     minutes           = absFloor(seconds / 60);
     hours             = absFloor(minutes / 60);
-    seconds %= 60;
+    seconds           = absFloor((seconds % 60) * 1000) / 1000;
```

Comment:
I also got another implementation of this formula but it work with `string` so I think the performance would be slower than the above implementation.

`seconds = parseFloat((seconds % 60).toFixed(3));`